java/test1/RedistTest.java

```diff
@@ -382,9 +386,20 @@ void parseLine(String line)
                         this.runCommands();
                     break;
                 case "profile":
-                    this.exitCode = this.api.profile();
-                    this.checkExitCode();
-                    this.checkSemicolon();
+                    if (rest.equals(" cpu on")) {
```

Comment:
I am starting to think that this code should be moved into a utility class for processing dat files - but I should do this later.

Reply:
Agreed, we are duplicating code and my changes to parsing code are hacky.